frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -145,8 +147,16 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
                 const filterParams = parsePeopleParams({ label, action, date_from, date_to, breakdown_value }, filters)
                 actions.setPeople(tempPeople)
                 people = await api.get(`api/person/stickiness/?${filterParams}${searchTermParam}`)
-            } else if (funnelStep) {
-                const params = { ...funnelLogic().values.filters, funnel_step: funnelStep }
+            } else if (funnelStep || filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+                let params
+                if (filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+                    // funnel trends
+                    const entrance_period_start = dayjs(date_from).format('YYYY-MM-DD HH:mm:ss')
+                    params = { ...funnelLogic().values.filters, entrance_period_start, drop_off: false }
```

Comment:
Why not use destructuring syntax and `useValues`?

Reply:
Same for :158 below

Reply:
`useValues`? like with kea's connect? 😮 or can you give me an example of what you mean with `useValues` in a logic file